pkgs/by-name/dh/dhcpig/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhc-pig";
+  version = "1.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "kamorin";
+    repo = "DHCPig";
+    rev = version;
+    hash = "sha256-MquLChDuJe3DdkxxKV4W0o49IIt7Am+yuhdOqUqexS8=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+    python3.pkgs.scapy
+  ];
+
+  buildInputs = [ ];
+
+  propagatedBuildInputs = [
```

```suggestion
  propagatedBuildInputs = with python3.pkgs; [
```
The same as above, just a small nitpick.